scripts/wrappers/control-plane-kicker.py

```diff
@@ -44,11 +45,43 @@ def stop_control_plane_services():
             set_service_expected_to_start(service, False)
 
 
+def microk8s_group_exists():
+    """
+    Check the existence of the microk8s group
+    :return: True is the microk8s group exists
+    """
+    try:
+        cmd = "getent group microk8s"
+        subprocess.check_call(cmd.split())
+        return True
+    except:
```

Comment:
Maybe add `CalledProcessError` here?  I could see this catching things it shouldn't.

Reply:
Done!